### PR TITLE
style: reword prompts

### PR DESCRIPTION
**Pull request by ghost**
## Description

Minor rewording of the installation script prompts

Signed-off-by: Lalith Vipulananthan [lalith.vipulananthan@tier4.jp](mailto:lalith.vipulananthan@tier4.jp)

## Pre-review checklist for the PR author

The PR author **must** check the checkboxes below when creating the PR.

- [x] I've confirmed the [contribution guidelines].
- [x] The PR follows the [pull request guidelines].

## In-review checklist for the PR reviewers

The PR reviewers **must** check the checkboxes below before approval.

- [x] The PR follows the [pull request guidelines].

## Post-review checklist for the PR author

The PR author **must** check the checkboxes below before merging.

- [x] There are no open discussions or they are tracked via tickets.

After all checkboxes are checked, anyone who has write access can merge the PR.

[contribution guidelines]: https://autowarefoundation.github.io/autoware-documentation/main/contributing/
[pull request guidelines]: https://autowarefoundation.github.io/autoware-documentation/main/contributing/pull-request-guidelines/

### CODE REVIEW

File: setup-dev-env.sh [hunk 1]
```diff
@@ -46,8 +46,8 @@ ansible_args=()
 if [ "$option_yes" = "true" ]; then
     echo -e "\e[36mRun the setup in non-interactive mode.\e[m"
 else
-    echo -e "\e[33mSetting up the build environment take up to 1 hour.\e[m"
-    read -rp ">  Are you sure to run the setup? [y/N] " answer
+    echo -e "\e[33mSetting up the build environment can take up to 1 hour.\e[m"
+    read -rp ">  Are you sure you want to run setup? [y/n] " answer
```
Comment (kenji-miyake) [inline setup-dev-env.sh]: `N` had the meaning of the default. Do you want to replace it with `n`?

Comment (ghost) [reply setup-dev-env.sh]: So, if the user just hits Enter, then "N" is assumed and setup will be cancelled? 
To revert the change, can I just make another commit to change it back to the original `N`?

Comment (kenji-miyake) [reply setup-dev-env.sh]: > So, if the user just hits Enter, then "N" is assumed and setup will be cancelled?

Yes.

FYI, other tools such as `apt` use the style. (In this case, the default is `Yes`.)

```
❯ sudo apt purge python3-rocker
Reading package lists... Done
Building dependency tree
Reading state information... Done
The following packages were automatically installed and are no longer required:
  python3-docker python3-websocket
Use 'sudo apt autoremove' to remove them.
The following packages will be REMOVED:
  python3-rocker*
0 upgraded, 0 newly installed, 1 to remove and 62 not upgraded.
After this operation, 77.8 kB disk space will be freed.
Do you want to continue? [Y/n]
```

> To revert the change, can I just make another commit to change it back to the original N?

Sure!

Comment (kenji-miyake) [reply setup-dev-env.sh]: @LalithVipulananthan I reverted it. :bow:

**Review by kenji-miyake — APPROVED** [review]

**Pull request merged**